### ISSUE TITLE
In Grapher: allow for columns for source and plot to be resized (widened)

### ISSUE THREAD

**Issue by miglto** [adamerose/PandasGUI#112]
Oftentimes I have columns that are too wide to be read fully in the space provided. Tried dragging the edge but that doesn't work. Thx.

**Comment by adamerose** [comment]
> Oftentimes I have columns that are too wide to be read fully in the space provided

Yeah it would also be good to improve the default sizing so things fit better without any resizing.

> Tried dragging the edge but that doesn't work.

Don't have this issue myself. Are you just unable to resize the columns in the Grapher? What about the nav bar on the left?

**Comment by miglto** [comment]
The main dataframe list window (all the way to the left) is resizable in my case. However, when I go to Grapher, the two columns within Grapher cannot be widened. Since some of my column names are fairly verbose for clarity, this is a problem.

**Comment by adamerose** [comment]
> The main dataframe list window (all the way to the left) is resizable in my case

Just to clarify - I'm asking if you can resize the individual columns in that, not the whole thing

![image](https://user-images.githubusercontent.com/10884874/109752782-08ad4b00-7baf-11eb-9591-bee407f3cb27.png)

**Comment by miglto** [comment]
Ok understood: I can resize each column in all sections of the GUI, and I can resize the width of the entire section on the left (which has the list of dataframes). But when I go to Grapher, I cannot resize the whole section where the columns is shown (in the "middle" I suppose) or the graph parameters on the right of that.

**Comment by miglto** [comment]
![109744320-127b8200-7ba0-11eb-9abd-d336f2cdf8e2](https://user-images.githubusercontent.com/51788468/109807506-fd304300-7bf3-11eb-95a6-7de21348a3d3.jpeg)

**Comment by miglto** [comment]
Once the tiny font problem was resolved (switching to scaling=100%) the width of the Grapher windows got more reasonable. They still cannot be resized but at least it is much better.
